keps/4803-workload-level-wait-for-pods-ready/README.md

```
@@ -0,0 +1,220 @@
+    - [Workload](#workload)
+    - [Jobs / Jobframework](#jobs--jobframework)
+  - [Webhooks](#webhooks)
+    - [Workload](#workload-1)
+    - [Jobs](#jobs)
+  - [Test Plan](#test-plan)
+    - [Prerequisite testing updates](#prerequisite-testing-updates)
+    - [Integration tests](#integration-tests)
+  - [Graduation Criteria](#graduation-criteria)
+- [Implementation History](#implementation-history)
+- [Drawbacks](#drawbacks)
+- [Alternatives](#alternatives)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)